src/components/form/Dropdown.vue

```diff
@@ -29,7 +29,7 @@ export default {
     props: {
         modelValue: {
             default: null,
-            type: String
+            type: [Number, String, Boolean]
```

Comment:
Boolean in a dropdown?

Reply:
very god point, didn't think of it like that, but there were instances of dropdowns being used with Boolean values - but you're absolutely right, they shouldn't be really...

Reply:
whilst our UX going forward may dictate Boolean values should use a checkbox (as it should) for now, it _is_ a valid value